### ISSUE TITLE
`try-except-pass` does not flag `contextlib.suppress(Exception)`

### ISSUE THREAD

**Issue by tjkuson** [charliermarsh/ruff#4923]
According to the [Python documentation](https://docs.python.org/3/library/contextlib.html#contextlib.suppress), this

```python
import contextlib

with contextlib.suppress(Exception):
    1 / 0
```

is equivalent to

```python
try:
    1 / 0
except Exception:
    pass
```

`try-except-pass` identifies flags the second code example, but not the first.

The the Python documentation says that the two code snippets are the same and the `suppressible-exception` autofixes the second code block to look like the first. Therefore, I think the `try-except-pass` rule should also flag `contextlib.suppress(Exception)` and recommend logging the exception in the first code example.
